src/messages.cpp

```diff
@@ -90,93 +90,96 @@ struct game_message : public JsonDeserializer, public JsonSerializer {
     }
 };
 
-bool message_exceeds_ttl(const game_message &message) {
-    return message_ttl > 0 && message.timestamp_in_user_actions + message_ttl <= g->get_user_action_counter();
+bool message_exceeds_ttl( const game_message &message )
+{
+    return message_ttl > 0 &&
+           message.timestamp_in_user_actions + message_ttl <= g->get_user_action_counter();
 }
 
 } //namespace
 
-class Messages::impl_t {
-public:
-    std::deque<game_message> messages;   // Messages to be printed
-    int                      curmes = 0; // The last-seen message.
-
-    bool has_undisplayed_messages() const {
-        return !messages.empty() && messages.back().turn() > curmes;
-    }
-
-    game_message const& history(int const i) const {
-        return messages[messages.size() - i - 1];
-    }
+class Messages::impl_t
+{
+    public:
+        std::deque<game_message> messages;   // Messages to be printed
+        int                      curmes = 0; // The last-seen message.
```

Comment:
I know you just changed the indent, but what's up with this huge space to make the `//` line up in this particular way? Shows up again down in display_messages() to line up the `=`.